jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -753,14 +753,19 @@ def _update_viewer_bound_steps(self, msg={}):
 
         viewer = self.viewer.selected_obj[0] if self.viewer_multiselect else self.viewer.selected_obj # noqa
         if not isinstance(viewer.state, ImageViewerState):
+            # We round these values to show, e.g., 7.15 instead of 7.1499999

```

Comment:
this also limits the ability to zoom in to small wavelength ranges... I'm not sure that is worth it

Reply:
maybe `decimals` can be based on the current range instead?

Reply:
Changing this to depend on the difference between min and max rather than just max makes sense to me.

Reply:
That also prevents decreasing `v_max` to be less than `v_min`, which is nice.